pom.xml

```diff
@@ -28,6 +28,7 @@
     <modules>
         <module>okta-hosted-login</module>
         <module>custom-login</module>
+        <module>self-hosted</module>
```

Comment:
What is the difference between "self-hosted" and "custom-login"

Reply:
Great question! They are one and the same technically. There is some confusion on naming the new samples that would use the Interaction Code grant (OIE) and I'm not sure if I need to rename `custom-login` to `self-hosted` or host the new samples in a separate folder named `self-hosted` login. I'll clarify that with PM.

Reply:
+100 for renaming, both use the widget in a self-hosted way.  Does the original example work if you tweak the sign-in widget config?